internal/pkg/runtime/launcher/native/launcher_linux.go

```diff
@@ -1156,18 +1157,20 @@ func squashfuseMount(ctx context.Context, img *imgutil.Image, imageDir string) (
 	}
 	sylog.Infof("Mounting SIF with FUSE...")
 
-	squashfusePath, err := bin.FindBin("squashfuse")
+	f, err := sif.LoadContainerFromPath(img.Path, sif.OptLoadWithFlag(os.O_RDONLY))
 	if err != nil {
-		return fmt.Errorf("squashfuse is required: %w", err)
+		return fmt.Errorf("failed to load image: %w", err)
 	}
-	if _, err := bin.FindBin("fusermount"); err != nil {
-		return fmt.Errorf("fusermount is required: %w", err)
+	defer func() { _ = f.UnloadContainer() }()
```

Comment:
Not sure that we want to do this here. `f` will be an open file handle to the SIF outside of any namespaces etc. This clearly shows that it is 'in use' by singularity, keeps any automounter happy etc.

Arguably we would actually benefit from performing the mount against the file handle that's open at this point... to remove the chance of the SIF being moved between here and the FUSE mount call.